docs/en/latest/config.json

```diff
@@ -60,10 +60,13 @@
         "concepts/apisix_route",
         "concepts/apisix_upstream",
         "concepts/apisix_tls",
-        "concepts/apisix_cluster_config",
-        "concepts/annotations"
+        "concepts/apisix_cluster_config"
       ]
     },
+    {
+      "type": "doc",
+      "id": "annotations"
+    },
```

Comment:
thanks for your contribution

But I think annotations are more appropriate in concepts.
Is there any reason for you to separate it into a separate page?

Reply:
Annotations are used in Kubernetes native Ingress resource while rest of the concepts is specific to APISIX's CRDs. IMO, annotations are a different entity. And it does not belong together with the concepts. I'm just thinking this out loud. Would like to hear your thoughts on this as well.

Reply:
Although annotations are a concept in Kubernetes, the configuration items that users can use are specific to APISIX Ingress. 
Users need learn these "concepts" when using this project, 
and we don't need to introduce the Ingress resource, because it is an existing concept in Kubernetes, and there is nothing special in the APISIX Ingress project,
what do you think?

Reply:
Okay. That kinda makes sense. I will move the page back.

Reply:
thanks